docs/en/interfaces/formats.md

```diff
@@ -515,11 +515,11 @@ Example:
 
 ## JSONAsString {#jsonasstring}
 
-In this format, a single JSON object is interpreted as a single value. If the input has several JSON objects (comma separated) they will be interpreted as separate rows.
+In this format, a single JSON object is interpreted as a single value. If the input has several JSON objects (comma separated) they will be interpreted as separate rows. The input data can be enclosed in an array of the format with square brackets.
```

Comment:
```suggestion
In this format, a single JSON object is interpreted as a single value. If the input has several JSON objects (comma separated) they will be interpreted as separate rows. If the input data is enclosed in square brackets, it will be interpreted as an array.
```